backend/app/controllers/api/v1/open_calls_controller.rb

```diff
@@ -0,0 +1,29 @@
+module API
+  module V1
+    class OpenCallsController < BaseController
+      include API::Pagination
+
+      def index
+        open_calls = OpenCall.all.includes(:investor)
```

Comment:
I see you don't use any gem for authorization -- recommend to check for example cancancan. Super easy to add at beginning of project almost seamlessly (no extra work) and it is super easy to define rights later on if you ever need them

Reply:
Not a fan on cancancan, more a pundit person ;] but that's a decision for later.

Reply:
Haha :), more cancancan person than pundit -- you have one simple file for all rights even for bigger projects with cancancan, but you have soo many files with rights with pundit